tests/getVisitsFilteredBy.test.js

```diff
@@ -149,34 +149,34 @@ test('Sorts by field', async (t) => {
   try {
     await rebuild();
 
-    const allResults = await visitorsFiltered(2);
+    const allResults = await visitorsFiltered(1);
     const sortedByYear = [...allResults].sort(
       (a, b) => a.yearofbirth > b.yearofbirth,
     );
 
-    const sortByYear = await visitorsFiltered(2, {
+    const sortByYear = await visitorsFiltered(1, {
       orderBy: 'yearofbirth',
     });
 
-    const sortWithFilter = await visitorsFiltered(2, {
+    const sortWithFilter = await visitorsFiltered(1, {
       orderBy: 'activity',
       filterBy: ['gender@male'],
     });
-    const filterByMale = await visitorsFiltered(2, {
+    const filterByMale = await visitorsFiltered(1, {
       filterBy: ['gender@male'],
     });
 
-    const sortByBadValue = await visitorsFiltered(2, {
+    const sortByBadValue = await visitorsFiltered(1, {
       orderBy: 'badvalue',
     });
 
     t.deepEqual(sortByYear, sortedByYear, 'Sorts by year');
```

Comment:
This test here is not working. I'm getting back what I want however the 'sortedByYear' function seems to just be returning the results in a random order rather than sorted and I'm not sure why. So the 'actual is correct however the 'expected' is not.